src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/Migration.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Bridge\Doctrine\Transport;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Types\Type;
+use Doctrine\DBAL\Types\Types;
+
+final class Migration
+{
+    private $tableName;
+
+    public function __construct(string $tableName = 'messenger_messages')
+    {
+        $this->tableName = $tableName;
+    }
+
+    public static function up(Schema $schema, Connection $connection): void
+    {
+        $useDeprecatedConstants = !class_exists(Types::class);
+        $table = $schema->createTable($this->tableName);
+        $table->addColumn('id', $useDeprecatedConstants ? Type::BIGINT : Types::BIGINT)->setAutoincrement(true)->setNotnull(true);
```

Comment:
It is not better a uuid ? :thinking:

Reply:
This isn't new code - it's already an int - iirc UUID's don't work super well with MySQL.

Reply:
Can we have a configuration for that  :/

Reply:
Of course! Open a pull request ;) Or use a different transport that you like better